hudi-common/src/main/java/org/apache/hudi/common/config/HoodieTableServiceManagerConfig.java

```diff
@@ -0,0 +1,187 @@
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".actions")
+      .defaultValue("")
+      .withDocumentation("Which action deploy on table service manager such as compaction:clean, default null");
+
+  public static final ConfigProperty<String> TABLE_SERVICE_MANAGER_DEPLOY_USERNAME = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".deploy.username")
+      .defaultValue("default")
+      .withDocumentation("The user name to deploy for table service of this table");
+
+  public static final ConfigProperty<String> TABLE_SERVICE_MANAGER_DEPLOY_QUEUE = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".deploy.queue")
+      .defaultValue("default")
+      .withDocumentation("The queue to deploy for table service of this table");
+
+  public static final ConfigProperty<String> TABLE_SERVICE_MANAGER_DEPLOY_RESOURCE = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".deploy.resource")
+      .defaultValue("4g:4g")
+      .withDocumentation("The resource to deploy for table service of this table, default driver 4g, executor 4g");
+
+  public static final ConfigProperty<Integer> TABLE_SERVICE_MANAGER_DEPLOY_PARALLELISM = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".deploy.parallelism")
+      .defaultValue(100)
+      .withDocumentation("The max parallelism to deploy for table service of this table, default 100");
+
+  public static final ConfigProperty<String> TABLE_SERVICE_MANAGER_DEPLOY_EXECUTION_ENGINE = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".execution.engine")
+      .defaultValue("spark")
+      .withDocumentation("The execution engine to deploy for table service of this table, default spark");
+
+  public static final ConfigProperty<String> TABLE_SERVICE_MANAGER_DEPLOY_EXTRA_PARAMS = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".deploy.extra.params")
+      .defaultValue("")
+      .withDocumentation("The extra params to deploy for table service of this table, split by ';'");
+
+  public static final ConfigProperty<Integer> TABLE_SERVICE_MANAGER_TIMEOUT = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".timeout")
+      .defaultValue(300)
+      .withDocumentation("Connection timeout for client");
+
+  public static final ConfigProperty<Integer> TABLE_SERVICE_MANAGER_RETRIES = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".connect.retries")
+      .defaultValue(3)
+      .withDocumentation("Number of retries while opening a connection to table service manager");
+
+  public static final ConfigProperty<Integer> TABLE_SERVICE_MANAGER_RETRY_DELAY = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".connect.retry.delay")
+      .defaultValue(1)
+      .withDocumentation("Number of seconds for the client to wait between consecutive connection attempts");
+
+  public static final ConfigProperty<String> RETRY_EXCEPTIONS = ConfigProperty
+      .key(TABLE_SERVICE_MANAGER_PREFIX + ".operation.retry.exceptions")
+      .defaultValue("IOException")
```

Comment:
shouldn't the class name be fully qualified?

Reply:
Will update this after retryHelper Improve pr merged.